packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -1654,7 +1654,9 @@ it('should render filter dropdown in the correct document', async () => {
     '.iui-filter-button',
   ) as HTMLElement;
   expect(filterToggle).toBeTruthy();
-  act(() => filterToggle.click());
+  act(() => {
+    filterToggle.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
```

Comment:
Change is needed because it now has `onMouseDown` handler instead of `onClick`. I can't use `fireEvent` or `userEvent` because they are throwing this error here: `It looks like the window object is not available for the provided node`

Reply:
Reverted.